.travis.yml

```diff
@@ -30,13 +31,19 @@ before_script:
   # point to fold-grammars source directory:
   - sed -i "s#rootDir = '/vol/fold-grammars/src/'#rootDir = '`pwd`/'#" Misc/Applications/lib/foldGrammars/Settings.pm
 script:
-  - if [[ "$TEST_SUBTASK" == "default" ]]; then
+  - if [[ "$TEST_SUBTASK" == "unambig" ]]; then
```

Comment:
the greater scope is co-folding, maybe a better name for this case than "unambig", since this is a big issue for many other parts of this repo :-)